app/tasks/s3.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import \
+    unicode_literals, \
+    absolute_import
+
+from . import celery
+from flask import current_app
+from io import BytesIO
+from os import getenv
+from app.api.services import brief_responses_service
+from app.models import BriefResponse
+from werkzeug.utils import secure_filename
+import boto3
+import botocore
+import zipfile
+
+
+class CreateResumesZipException(Exception):
+    """Raised when the resume zip fails to create."""
+
+
+@celery.task
+def create_resumes_zip(brief_id):
+    BUCKET_NAME = current_app.config.get('S3_BUCKET_NAME')
+    s3 = boto3.resource(
+        's3',
+        region_name=getenv('AWS_REGION'),
+        aws_access_key_id=getenv('AWS_ACCESS_KEY_ID'),
+        aws_secret_access_key=getenv('AWS_SECRET_ACCESS_KEY')
+    )
+    bucket = s3.Bucket(BUCKET_NAME)
+
+    files = []
+    brief_responses = brief_responses_service.get_brief_responses(brief_id, None)
+    for brief_response_simple in brief_responses:
+        brief_response = BriefResponse.query.get(brief_response_simple['id'])
```

Comment:
I think you should create a method in brief response service that returns all the data needed to do this operation instead of making separate queries in a for loop. 

I would imagine the result looking something like this, where you would have 1 row per attachment_url:

brief_id  |  supplier_id  |  supplier_code  |  supplier_name  |  attachment_url